subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/inception/GradleBuildPerformanceTest.groovy

```diff
@@ -145,6 +149,8 @@ class GradleBuildPerformanceTest extends Specification {
         def eagerBuildName = 'eager build'
         def lazyBuildName = 'lazy build'
 
+
+
```

Comment:
Nit: I think these two blank lines can be removed.

Reply:
Removed.